src/styles/index.scss

```
@@ -1,3 +1,132 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$blue-color: #00acdc;
+
+@mixin space-between-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-bottom: 16px;
+
+  }
+
+  &__review {
+    @include space-between-content;
+    margin-bottom: 24px;
+
+    &-count {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    @include space-between-content;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $blue-color;
+    border: 1px solid;
+    border-color: #00acdc;
+    border-radius: 5px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    &:hover {
+      border-color: $blue-color;
+      background-color: #fff;
+      color: $blue-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    margin-left: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  & :first-child {
+    margin-left: 0;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

add cases for all stars to make it flexible
```suggestion
  &--0
  .....
  &--4 :nth-child(-n + 4) {
    background-image: url(../images/star-active.svg);
  }
  
```

---

all updated:)